server/routes/caseAdmin/initialChecks/check.ts

```diff
@@ -0,0 +1,53 @@
+import type { Request, Response } from 'express'
+import { validateRequest } from '../../../middleware/setUpValidationMiddleware'
+import paths from '../../paths'
+import type { EligibilityAndSuitabilityService } from '../../../services'
+
+export default class CheckRoutes {
+  constructor(private readonly eligibilityAndSuitabilityService: EligibilityAndSuitabilityService) {}
+
+  GET = async (req: Request, res: Response): Promise<void> => {
+    const { checkCode, type, prisonNumber } = req.params
+
+    const { assessmentSummary, check } = await this.eligibilityAndSuitabilityService.getInitialCheck(
+      req?.middleware?.clientToken,
+      prisonNumber,
+      type as 'eligibility' | 'suitability',
+      checkCode,
+    )
+
+    res.render('pages/caseAdmin/initialChecks/check', { assessmentSummary, type, check })
+  }
+
+  POST = async (req: Request, res: Response): Promise<void> => {
+    const { checkCode, type, prisonNumber } = req.params
+
+    const { check, nextCheck } = await this.eligibilityAndSuitabilityService.getInitialCheck(
+      req?.middleware?.clientToken,
+      prisonNumber,
+      type as 'eligibility' | 'suitability',
+      checkCode,
+    )
+
+    validateRequest(req, body =>
+      check.questions.flatMap(question => {
+        const value = body?.[question.name]
+        const missingValue = value === undefined || (value !== 'true' && value !== 'false')
+        return missingValue ? [{ field: question.name, message: question.text }] : []
+      }),
+    )
```

Comment:
performing validation.. for each question in this check make sure it's been answered